code/__DEFINES/~skyrat_defines/logging.dm

```diff
@@ -1,12 +1,11 @@
 // Logging types for log_message()
-#define LOG_SUBTLER (1 << 20)
-#define LOG_AMBITION (1 << 21)
+#define LOG_SUBTLER (1 << 22)
```

Comment:
Why 22 and not 20?

Reply:
20 is victim logs which is what caused the issues with attacks being logged as subtler emotes, and I forgot what 21 is but it's something